swupd/manifest.go

```diff
@@ -767,7 +767,7 @@ func constructIndex(c *config, ui *UpdateInfo, f2b []*bundleIndex) error {
 // sorts the index first by filename then by bundle name. writeIndexManifest creates a new
 // bundle in which the file will live. This bundle is added to the "full" manifest which
 // is part of the bundles slice. A path string to the new manifest is returned on success.
```

Comment:
Update comment to Manifest pointer is returned.

Reply:
ah, thanks